tutorial/05 - Presentation Layouts.ipynb

```diff
@@ -383,7 +379,7 @@
    "metadata": {},
    "source": [
     "# Rows and Columns\n",
-    "The `bokeh.layouts` modules provides the ``row`` and ``column`` functions to arrange plot objects in vertical or horizontal layouts. Below is an example of three plots arranged in a row."
+    "The `bokeh.layouts` modules provide the ``row`` and ``column`` functions to arrange plot objects in vertical or horizontal layouts. Below is an example of three plots arranged in a row."
```

Comment:
`bokeh.layouts` is a single module, so the proper fix here is to un-pluralize "modules":
> The `bokeh.layouts` module provides the